src/olympia/devhub/views.py

```diff
@@ -1703,8 +1709,18 @@ def request_review(request, addon_id, addon):
     if not addon.can_request_review():
         return http.HttpResponseBadRequest()
 
-    addon.update(status=amo.STATUS_NOMINATED)
-    messages.success(request, _('Review Requested.'))
+    latest_version = addon.find_latest_version_including_rejected(
+        amo.RELEASE_CHANNEL_LISTED)
+    if latest_version:
+        latest_version.files.filter(
+            status=amo.STATUS_DISABLED).update(
+            status=amo.STATUS_AWAITING_REVIEW)
+    if addon.has_complete_metadata():
+        addon.update(status=amo.STATUS_NOMINATED)
+        messages.success(request, _('Review Requested.'))
+    else:
+        messages.success(request, _(
+            'Review Requested.  You must provide further details to proceed.'))
```

Comment:
When you click 'request review' in devhub versions and status page - it shows up as a banner at the top of the details page.